wire-ios-transport/Source/TransportSession/ZMTransportSession.m

```diff
@@ -503,9 +503,6 @@ - (void)didCompleteRequest:(ZMTransportRequest *)request data:(NSData *)data tas
     [self.remoteMonitoring logWithResponse:httpResponse];
 
     ZMLogDebug(@"ConnectionProxyDictionary: %@,", session.configuration.connectionProxyDictionary);
-    ZMLogPublic(@"Response to %@: %@", request.safeForLoggingDescription,  response.safeForLoggingDescription);
-    ZMLogInfo(@"<---- Response to %@ %@ (status %u): %@", [ZMTransportRequest stringForMethod:request.method], request.path, (unsigned) httpResponse.statusCode, response);
-    ZMLogInfo(@"URL Session is %@", session.description);
```

Comment:
here I think we have the safe correct info in DataDog. so it's ok to just delete 👍